### PR TITLE
complete a previous file renaming

### PR DESCRIPTION
**Pull request by pberthonneau**
complete unfinished commit 174a4f700f39c0fad2ad01411c8f6e845ea09831

### CODE REVIEW

File: examples/basic/README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-```
+```bash
 pipenv install -d
 pipenv shell
 export GH_TOKEN=<your github API token>
 
-python examples/basic/http-endpoint.py $GH_TOKEN profusion/sgqlc
+cd examples/basic
+python 01_http_endpoint.py $GH_TOKEN profusion/sgqlc
```
Comment (barbieri) [inline examples/basic/README.md]: would you please use `python3` just to make sure people get it right? not every OS has `python`  symlinked to the `python3` (in all code examples)

Comment (pberthonneau) [reply examples/basic/README.md]: sure

**Comment by barbieri** [comment]
looks good, could you merge (squash) the commits into a single one? Otherwise I'll do it and merge (fast-forward with clean commits only in this repo)

**Comment by pberthonneau** [comment]
First time I do this! Looks ok locally, but tell me if this good for you.

**Comment by barbieri** [comment]
that's it ;-)

thanks!

**Pull request merged**